numpy/core/tests/test_multiarray.py

```diff
@@ -4714,6 +4714,10 @@ def test_empty_files_text(self):
         y = np.fromfile(self.filename, sep=" ")
         assert_(y.size == 0, "Array not empty")
 
+    def test_tobytes_order(self):
+        x = np.array([[1, 2, 3], [4, 5, 6]], order='F')
+        assert_(x.tobytes('F') == x.tobytes('K'))
```

Comment:
This will give a wrong result as LHS = 2 whereas the RHS = 3.
When the order of 'x' will be in K, then the tobytes function should hold.